Source/DafnyLanguageServer.Test/Various/ShutdownTest.cs

```diff
@@ -158,14 +157,14 @@ public class ShortLivedProcessStarter {{
         "System.ComponentModel.Primitives"
       };
       compilation = compilation.AddReferences(standardLibraries.Select(fileName =>
-        MetadataReference.CreateFromFile(Assembly.Load(fileName).Location)));
-      compilation = compilation.WithOptions(new CSharpCompilationOptions(OutputKind.ConsoleApplication))
-        .AddReferences(
-          MetadataReference.CreateFromFile(typeof(object).GetTypeInfo().Assembly.Location),
-          MetadataReference.CreateFromFile(Assembly.Load("mscorlib").Location));
+            MetadataReference.CreateFromFile(Assembly.Load(fileName).Location))).
+        AddReferences(
+            MetadataReference.CreateFromFile(typeof(object).GetTypeInfo().Assembly.Location),
+            MetadataReference.CreateFromFile(Assembly.Load("mscorlib").Location));
+      compilation = compilation.WithOptions(new CSharpCompilationOptions(OutputKind.ConsoleApplication));
```

Comment:
You could continue the method chaining here since you neither explicitly name nor use the intermediate state.